auslib/web/admin/swagger/api.yaml

```diff
@@ -0,0 +1,423 @@
+                example: ["balrogadmin"]
+  /rules:
+    get:
+      summary: Returns filtered list of Rules in Balrog's DB
+      description: "Returns filtered list Rules in Balrog’s database
+      inside of a JSON Object. Returns all rules if 'product' query parameter isn't passed"
+      tags:
+        - Rules
+      operationId: auslib.web.admin.views.mapper.rules_get
+      parameters:
+        - name: product
+          in: query
+          description: Name of the product to filter the rules against.
+          type: string
+          x-nullable: true
+          required: false
+        - name: format
+          in: query
+          description: json format applied during testing.
+          type: string
+          x-nullable: true
+          required: false
+      produces: [application/json]
+      deprecated: false
+      responses:
+        '200':
+          description: Get all Rules
+          schema:
+            type: object
+            required:
+              - count
+              - rules
+            properties:
+              count:
+                description: count of total rules fetched
+                type: integer
+                format: int32
+                example: 1
+              rules:
+                description: array where each element is a rule object
+                type: array
+                items:
+                  $ref: '#/definitions/RulesBaseGET'
+                example:
+                  - alias: "esr45-localtest"
+                    backgroundRate: 100
+                    buildID: null
+                    buildTarget: null
+                    channel: "esr-localtest"
+                    comment: "esr45 esr-localtest rule"
+                    data_version: 22
+                    distVersion: null
+                    distribution: null
+                    fallbackMapping: null
+                    headerArchitecture: null
+                    locale: null
+                    mapping: "Firefox-45.7.0esr-build1"
+                    osVersion: null
+                    priority: 43
+                    product: "Firefox"
+                    rule_id: 308
+                    systemCapabilities: null
+                    update_type: "minor"
+                    version: null
+                    whitelist: null
+    post:
+      tags:
+        - Rules
+      summary: creates a Rule
+      description: creates a new Rule in Balrog's DB
+      consumes:
+        - application/json
+      produces:
+        - text/html
+        - application/json
+      deprecated: false
+      operationId: auslib.web.admin.views.mapper.rules_post
+      parameters:
+        - name: rule
+          in: body
+          description: New Rule object data
+          required: true
+          schema:
+            allOf:
+              - $ref: '#/definitions/CSRFModel'
+              - $ref: '#/definitions/RulesBase'
+
+            #If you change the order, make sure to modify test cases too.
```

Comment:
about the "format" paramter above, I forgot to delete it from swagger spec. Already modified the test cases to no longer use it anymore.